lms/djangoapps/courseware/views/views.py

```diff
@@ -872,6 +872,7 @@ def program_marketing(request, program_uuid):
 
     if program.get('is_learner_eligible_for_one_click_purchase') and skus:
         context['buy_button_href'] = ecommerce_service.get_checkout_page_url(*skus)
+        context['buy_button_href'] += '&bundle={}'.format(program_uuid)
```

Comment:
How about pushing this into `get_checkout_page_url` and passing the program UUID as a kwarg?

Reply:
That makes more sense, will fix.